src/test/java/in/zapr/druid/druidry/granularity/PeriodGranularityTest.java

```diff
@@ -0,0 +1,46 @@
+package in.zapr.druid.druidry.granularity;
```

Comment:
Please add license in the file

Reply:
We do not have license in any of the Test file. Why would it be needed?